webClient/src/app/editor/code-editor/monaco/monaco.service.ts

```diff
@@ -87,29 +143,11 @@ export class MonacoService {
         }
       });
     } else {
-      let requestUrl: string;
-      let filePath = ['/', '\\'].indexOf(fileNode.model.path.substring(0, 1)) > -1 ? fileNode.model.path.substring(1) : fileNode.model.path;
-      let _observable;
-
-      if (reload) {
-        if (fileNode.model.isDataset) {
-          requestUrl = ZoweZLUX.uriBroker.datasetContentsUri(filePath);
-          _observable = this.http.get(requestUrl).map((res: any) => this.dataAdapter.convertDatasetContent(res._body));
-        } else {
-          requestUrl = ZoweZLUX.uriBroker.unixFileUri('contents',
-                                                      filePath+'/'+fileNode.model.fileName,
-                                                      { responseType: 'b64' });
-          _observable = this.http.get(requestUrl).map((res: any) => this.dataAdapter.convertFileContent(res._body));
-        }
-
-      } else {
-        _observable = new Observable((obs) => obs.next({ contents: fileNode.model.contents }));
```

Comment:
@DivergentEuropeans This line removed your PR. When `reload` is `false` we don't want to reload the file content.

Reply:
Yep